src/twisted/topfiles/8900.feature

```diff
@@ -0,0 +1,2 @@
+Added twisted.internet.defer.unbox function to enable the unboxing of a
+"resolved" Deferred.
```

Comment:
Seems like yet a 3rd language here on what we call "a deferred that's already fired".